Translations/ja.lproj/FinderExtensionLocalizable.strings

```diff
@@ -1,11 +1,11 @@
 "Open with Keka" = "Kekaで開く";
 "Compress here" = "ここに圧縮";
-"Compress to..." = "圧縮先・・・";
-"Compress the contents of the folder here" = "ここにあるフォルダの中身を圧縮する";
-"Compress the contents of the folder to..." = "フォルダの中身を圧縮して...";
-"Encrypt with the default password here" = "ここでデフォルトのパスワードで暗号化する";
-"Encrypt with the default password to..." = "デフォルトのパスワードで暗号化して...";
+"Compress to..." = "圧縮先...";
+"Compress the contents of the folder here" = "ここにフォルダを圧縮する";
+"Compress the contents of the folder to..." = "フォルダを圧縮...";
```

Comment:
Here is important the "contents of the folder" since this action does not compress the folder but its contents. Not sure If the new translation makes it clear.

Reply:
I understand the behavior
I'll fix this string soon